app/models/types_de_champ/linked_drop_down_list_type_de_champ.rb

```diff
@@ -92,6 +96,35 @@ def columns(procedure_id:, displayable: true, prefix: nil)
 
   private
 
+  def add_blank_option_when_not_mandatory(options)
+    return options if mandatory
+    options.unshift('')
+  end
+
+  def get_primary_value(champ)
+    if champ.value.present?
+      JSON.parse(champ.value)[0]
+    else
+      ''
+    end
+  rescue JSON::ParserError
+    ''
+  end
```

Comment:
ca m'embete qu'on rajoute encore une pile de `JSON.parse` au lieu de migrer nativement les values dans le `value_json` mais si on doit aller dans ce chemin, proposition d'écriture plus light

- j'enlève le `get` parce qu'on fait pas du java ici :)
- je retourne `nil` parce qu'on est typé (? je suis pas sur de moi)

```suggestion
  def primary_value(champ) = champ.value&.then { JSON.parse(_1)[0] rescue nil }
```

Reply:
Je suis d'accord avec toi, mais pour le coup, je n'ai pas envie de mettre de l'énergie maintenant à migrer le champ linked_drop_down. Mais oui, il faut qu'on parle de ce champ.